prebuilt_swift_static_framework/prebuilt_swift_static_framework.bzl

```diff
@@ -86,24 +125,53 @@ _prebuilt_swift_static_framework = rule(
             cfg = "host",
             executable = True,
         ),
-    ),
+    )),
+    fragments = ["apple"],
     outputs = {
         "fat_file": "%{name}.fat",
         "output_file": "%{name}.zip",
     },
 )
 
-def prebuilt_swift_static_framework(name, srcs = [], deps = []):
+def prebuilt_swift_static_framework(name, srcs = [], deps = [], **kwargs):
```

Comment:
Instead of `kwargs`, should the exact parameters be declared? As it is, the caller could specify a keyword argument that doesn't get used. Or, the body of this could fail if there are extra unused `kwargs`.

Reply:
Updated the documentation to list the parameters that are supported for this rule. If caller passes an unsupported keyword, it's ignored.

Let me know if you would still prefer to list all of the parameters with default values.